static/scripts/ocap.entity.js

```diff
@@ -81,9 +81,8 @@ class Entity {
 	}
 
 	createMarker(latLng) {
-		let marker = L.marker(latLng).addTo(map);
-		marker.setIcon(this._realIcon);
-		marker.setRotationOrigin(this._markerRotationOrigin);
+		let marker = L.marker(latLng, { icon: this._realIcon, rotationOrigin: this._markerRotationOrigin }).addTo(map);
+		// L.DomUtil.addClass(marker.getContainer(), "animation");
```

Comment:
This throws errors

Reply:
Wrap `this._realIcon` in parentheses, since it's a function call that needs to resolve in order to give the returned value as the option

Reply:
no need, it works fine